libraries/core/src/androidTest/kotlin/com/bumble/appyx/core/plugin/BackPressHandlerTest.kt

```diff
@@ -91,7 +91,7 @@ class BackPressHandlerTest {
         Espresso.onIdle()
         rule.waitForIdle()
 
-        assertThat(rule.activityScenario.result.resultCode, equalTo(Activity.RESULT_CANCELED))
```

Comment:
This test was failing with the following exception: `You must start Activity first. Make sure you are using launchActivityForResult() to launch an Activity.`.

Our tests are not calling `launchActivityForResult()`, hence why the result code no longer works. I believe checking that the activityScenario state is destroyed should be sufficient.